xapian-core/include/xapian/weight.h

```diff
@@ -590,6 +590,101 @@ class XAPIAN_VISIBILITY_DEFAULT BM25Weight : public Weight {
     double get_maxextra() const;
 };
 
+/// Xapian::Weight subclass implementing the BM25+ probabilistic formula.
+class XAPIAN_VISIBILITY_DEFAULT BM25PlusWeight : public BM25Weight {
+
+  public:
+    /** Construct a BM25PlusWeight.
+     *
+     *  @param delta  A parameter for pseudo tf value to control the scale
+     *		      of the tf lower bound. Delta(δ) can be tuned for e.g
+     *		      from 0.0 to 1.5 but BM25+ can still work effectively
+     *		      across collections with a fixed δ = 1.0. (default 1.0)
+     */
+    BM25PlusWeight(double k1, double k2, double k3, double b, double min_normlen, double delta)
+	: param_k1(k1), param_k2(k2), param_k3(k3), param_b(b),
+	  param_min_normlen(min_normlen), param_delta(delta)
```

Comment:
The documentation comment only covers parameter `delta`, and not any of the parameters which are in common with `BM25Weight` - to find out what they mean the user would have to guess to go and look at the documentation for that.  It would be better to just copy over the documentation for the common parameters (except that `k2` isn't actually used by BM25+).